### ISSUE TITLE
Wireframe of website layout

### ISSUE THREAD

**Issue by stevenmusumeche** [stevenmusumeche/asg-website#6]
Great a wireframe of the layout of the site.  No styling, just layout.

**Comment by davinthesmith** [comment]
Working this now. @arthyn let me know if you've already made some progress so we can sync up.

**Comment by davinthesmith** [comment]
Gotta head out.  I put my latest commit there if you want to pull from there.  I'm not making any final style choices (I purposefully picked ugly ones) until we have decided on final fonts and colors.

I tried the `Press Start 2P` font for headers... not a huge fan.  Feels really really big.  Might just need some tweaking? idk

In this commit:
- Added a few packages
- Reorganized plugins alphabetically
- Stripped out stuff we aren't using
- Added `styles` directory with typography and examples for color configurations
- Renamed page-2 to Events
- Removed gatsby references from footer and siteMetadata

Packages added
- Typography.js: https://www.gatsbyjs.org/packages/gatsby-plugin-typography
- StyledComponents.js: https://www.gatsbyjs.org/packages/gatsby-plugin-styled-components
